CMakeLists.txt

```diff
@@ -84,15 +84,7 @@ endif()
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 
-# Enable the built-in updater by default on the three main supported OS
-# unless environmental variable WITH_UPDATER is defined and set to "NO" (case
-# insensitive). Linux packagers will find it useful to do this since they are
-# likely to be responsible for updates there. Automatically exclude the
-# update stuff from other OSs such as FreeBSD and Cygwin because if they ever
-# get finished they have their own packaging system - and we do not support
-# updating them.
-# WITH_UPDATER is an environmental value/variable (so could be a number, a
-# string, something else or not even exist).
+# Set the environment variable WITH_UPDATER=NO to disable the built-in updater
```

Comment:
A further alteration I would suggest here and in the other places - is to separate the variable name from the value:
```
# Set the environment variable WITH_UPDATER to NO to disable the built-in updater
```

Reply:
I think this way is good so people have to spend less time thinking and understanding - they can just copy/paste this and go.